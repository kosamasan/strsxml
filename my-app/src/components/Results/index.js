import React from 'react'

const Results = ({data}) => {
    return <div>{data}</div>
}

export default Results