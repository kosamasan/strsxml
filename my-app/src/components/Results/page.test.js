import React from 'react'
import Results from './index'

it('renders succesfully', () => {
  const results = <Results />
  expect(results).toMatchSnapshot()
})