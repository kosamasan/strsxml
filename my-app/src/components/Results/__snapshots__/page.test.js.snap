// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders succesfully 1`] = `<Results />`;
